config/webpack.dev.config.js

```diff
@@ -4,6 +4,7 @@ const Merge = require('webpack-merge');
 const path = require('path');
 const webpack = require('webpack');
```

Comment:
Note to Reviewers: Changes in this file are for local testing only. Will be removed before merge.